tests/test_buildSingleKernelVisitor.py

```diff
@@ -9,22 +9,22 @@
 import lsst.utils.logging as logUtils
 import lsst.pex.config as pexConfig
 
+from lsst.ip.diffim import PsfMatchConfigDF
+
 logUtils.trace_set_at("lsst.ip.diffim", 4)
 
 
 class DiffimTestCases(unittest.TestCase):
 
     def setUp(self):
-        self.config = ipDiffim.ImagePsfMatchTask.ConfigClass()
-        self.config.kernel.name = "DF"
-        self.subconfig = self.config.kernel.active
+        self.config = PsfMatchConfigDF()
```

Comment:
Just for a one-time use, it doesn't seem worth importing `PsfMatchConfigDF` from `ip.diffim` when you are anyway importing the whole package. I'd recommend using `ipDiffim.PsfMatchConfigDF` here.